pkgs/top-level/qt6-packages.nix

```diff
@@ -27,7 +26,6 @@ makeScopeWithSplicing' {
     inherit (self) callPackage;
     noExtraAttrs = set: lib.attrsets.removeAttrs set [ "extend" "override" "overrideScope" "overrideDerivation" ];
   in (noExtraAttrs qt6) // {
-  inherit stdenv;
```

Comment:
Maybe we should add a warning here for a release cycle first. LGTM otherwise.

Reply:
I've reverted this change, since I'm not sure how to properly add a warning here...

Reply:
I think just `stdenv = lib.warn "qt6Packages.stdenv: this is going away!!" stdenv;` would do it, as long as there’s nothing else in the tree using it.

Reply:
Indeed, thanks. I wonder what I was worried about...